src/applications/personalization/profile/components/personal-information/ContactInformationField.jsx

```diff
@@ -253,20 +301,35 @@ class ContactInformationField extends React.Component {
           title={title}
         />
 
-        {this.isEditLinkVisible() && (
-          <button
-            aria-label={`Edit ${title}`}
-            type="button"
-            data-action="edit"
-            onClick={() => {
-              this.onEdit(isEmpty ? 'add-link' : 'edit-link');
-            }}
-            id={getEditButtonId(fieldName)}
-            className={classes.editButton}
-          >
-            Edit
-          </button>
-        )}
+        <div className="vads-u-display--flex vads-u-flex-wrap--wrap vads-u-flex-direction--column vads-u-width--full">
```

Comment:
all this flex stuff seems unnecessary since it only contains a single div as a child?